### PR TITLE
Loki and grafana deployment

### PR DESCRIPTION
**Pull request by pdittaro**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
-      - ./settings/keycloak/keycloak-local-realm.json:/tmp/keycloak-local-realm.json
-      - ./settings/keycloak/keycloak-local-user.sh:/tmp/keycloak-local-user.sh
-    ports:
-      - "8080:8080"
-      - "8443:8443"
-    networks:
-      - mds
-    # logging:
-    #     driver: none
+  # keycloak:
```
Comment (exzizt) [inline docker-compose.yml:5]: Any reason why this is commented out? Should it be removed?

File: docker-compose.yml [hunk 2]
```diff
@@ -220,14 +197,123 @@ services:
...
+    ports:
+      - 3010:3000
+    volumes:
+      - grafana_storage:/var/lib/grafana
+      - ./monitoring/grafana/provisioning/:/etc/grafana/provisioning/
+    env_file:
+      - ./monitoring/grafana/env
+    restart: always
+
+  # jaeger:
```
Comment (exzizt) [inline docker-compose.yml:251]: No jaegar?

Comment (pdittaro) [reply docker-compose.yml:251]: Not yet, next PR, didn't feel like removing the WIP though

File: docker-compose.yml (hunk 1 above)
Comment (pdittaro) [reply docker-compose.yml:5]: This was commented out as I felt like our docker-compose was getting pretty full. But seriously, we might need to consider a different approach for how we manage local services.

**Pull request merged**